fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/algebra/CatalogRelation.java

```diff
@@ -27,4 +27,9 @@ public interface CatalogRelation extends Relation {
     TableIf getTable();
 
     DatabaseIf getDatabase() throws AnalysisException;
+
+    // For empty table, nereids require getting 1 as row count. This is a wrap function for nereids to call getRowCount.
+    default long getRowCountForNereids() {
+        return Math.max(getTable().fetchRowCount(), 1);
```

Comment:
I think we should use getTable().getRowCount(), because for external table, getRowCount reads data in cache, fetchRowCount may connect remote data source, could be very heavy.

Reply:
GET